### PR TITLE
feat: Add `.eslintcache` to gitignore

### LINKED ISSUE #318: [FEATURE] [CHORE] Add `.eslintcache` to gitignore
### Description

This file contains nothing useful rather than cache, and it's causing unnecessary git errors/warnings. It would be better to add it to gitignore

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by ShatilKhan**
# Fixes #318 

**My PR closes #issue_number_here**

# 👨‍💻 Changes proposed
- Added  `.eslintcache` to gitignore file

# ✔️ Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

### CODE REVIEW

**Comment by ShatilKhan** [comment]
@Dun-sin please review

**Review by Dun-sin — CHANGES_REQUESTED** [review]
wrong place it should be in the client folder

File: .gitignore
Comment (Dun-sin) [inline .gitignore:1]: this file should be here at all

**Comment by ShatilKhan** [comment]
Done @Dun-sin 
Please review

File: .gitignore
Comment (ShatilKhan) [reply .gitignore:1]: Oh sorry about that, I'll remove it

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Comment by ShatilKhan** [comment]
Thank you so much @Dun-sin !
I'll make sure to stick around!

**Pull request merged**